doc/classes/SplitContainer.xml

```diff
@@ -30,7 +30,7 @@
 	</methods>
 	<members>
 		<member name="collapsed" type="bool" setter="set_collapsed" getter="is_collapsed" default="false">
-			If [code]true[/code], the area of the first [Control] will be collapsed and the dragger will be disabled.
+			If [code]true[/code], the dragger will be disabled and the children will be sized as if the [member split_offset] was [code]0[/code].
```

Comment:
Unrelated, please make a separate commit so this can be cherry picked (we don't cherry pick tests)

Reply:
- In that case, I should do the same with my https://github.com/godotengine/godot/pull/97255